gradle.properties

```diff
@@ -15,10 +15,10 @@ org.gradle.jvmargs=-Xmx1024M
 #
 # TODO(GA): This is currently a placeholder for crates we are going to stabilize at the time of GA.
 # Until then, a value of this key can contain 0 for the major version.
```

Comment:
Will separately create a PR against `smithy-rs-release-1.x.y` to clean up `TOOD(GA)` comments.